### PR TITLE
update manual mode docs to cover new upgrade behavior

### PR DESCRIPTION
**Pull request by joelddiaz**

### CODE REVIEW

File: docs/mode-manual-creds.md [hunk 1]
```diff
@@ -103,8 +85,22 @@ Finally, proceed with cluster creation:
...
-
 ## Upgrades
 
-In OpenShift 4.6+ we plan to add additional code to help prevent situations where a user may enter an upgrade that will fail because their manually maintained credentials have not been updated to match the CredentialsRequests in the upcoming release image. In the meantime we are monitoring to ensure no upgrade breaking credentials changes go in.
+It is important to note that before performing an upgrade from one minor version to the next (ie 4.7.x to 4.8.y), you may need to adjust your credentials if permissions have changed in the next release. The Cloud Credential Operator will mark itself Upgradeable=False when configured for Manual mode. The Upgradeable=False status should not affect z-stream upgrades (ie 4.7.4 to 4.7.5).
+
+Before upgrade from one minor version to the next, the cluster admin(s) should review the bundle of CredentialsRequest objects from the version being upgraded to. First extract the CredentialsRequest objects for the cluster's cloud:
+
+```bash
```
[the remark this answers is not in the document]
Comment (joelddiaz) [reply docs/mode-manual-creds.md:94]: Well...the CCO tooling only helps for STS (and just recently IBMCloud). This document predates ccoctl and covers the disconnected VPC case, the keep-admin-creds-out-of-the-cluster case, etc. I will add a mention to ccoctl with a "depending on your cluster credentials configuration, the ccoctl tool may help automate many of these steps...".

**Comment by joelddiaz** [comment]
updated to address PR feedback

File: docs/mode-manual-creds.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dgoodwin) [reply docs/mode-manual-creds.md:96]: Ok gotcha, we really should get that updated for manual mode too at some point

**Pull request merged**